TrailBlaze/Networking/PaceImprovementTracker.h

```diff
@@ -8,21 +8,27 @@
 #import <Foundation/Foundation.h>
 #import "CoreLocation/CoreLocation.h"
 #import "CoreMotion/CoreMotion.h"
+@import Parse;
 
 NS_ASSUME_NONNULL_BEGIN
 
 
 @interface PaceImprovementTracker : NSObject
 @property (strong, nonatomic) CMPedometer  *pedometer;
-@property (strong, nonatomic) NSMutableDictionary *currentPacesArray;
+@property (strong, nonatomic) NSMutableDictionary *currentPacesDictionary;
 @property (strong, nonatomic) NSDictionary *bestPacesDictionary;
 @property (strong, nonatomic) NSArray *polylinePoints;
+@property (strong, nonatomic) NSArray *nextPoints;
+@property int i;
+@property (strong, nonatomic) NSDate *startDate;
+@property (strong, nonatomic) PFObject *runObject;
```

Comment:
follow up from a comment in previous pr, are you planning to make these properties private (if possible)? definitely starting to leak some implementation details here